DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -15,6 +17,12 @@ public interface IRegistrationService
             string baseUrl,
             string userIp
         );
+
+        string RegisterAdminDelegate(
+            RegistrationModel registrationModel
+        );

```

Comment:
I'm a little surprised this is formatted like this when there are other much longer lines in the file.

Reply:
Also should this be public if the only place it seems to be called is another method in this service?